nuxt.config.ts

```diff
@@ -175,17 +175,27 @@ export default defineNuxtConfig({
     // https://v8.i18n.nuxtjs.org/getting-started/setup
     i18n: {
         strategy: 'prefix_except_default',
-        detectBrowserLanguage: false,
+        detectBrowserLanguage: {
+            useCookie: true,
+            redirectOn: 'all',
+        },
         defaultLocale: I18N_DEFAULT_LOCALE,
-        locales: I18N_LOCALES.map(locale => ({
-            code: locale,
-            file: `nuxt.${locale}.json`,
-        })),
+        locales: I18N_LOCALES.map((locale) => {
+            // Order impact overriding message
+            return [
+                {
+                    code: locale,
+                    file: `nuxt.${locale}.json`,
+                },
+                {
+                    code: locale,
+                    file: `nuxt.${locale}.default.json`,
```

Comment:
I don't think it's a good idea to split translations into multiple files. The default file won't be available into xilofone